addons/ammunition/CfgAmmo.hpp

```diff
@@ -48,4 +50,21 @@ class CfgAmmo {
         submunitionAmmo = QCLASS(12g_Pellets_Submunition_Deploy);
         submunitionConeAngle = 0.81;
     };
+    CLASS(556x45_EPR_M855A1): B_556x45_Ball {
+        airFriction=-0.0012588;
+        ACE_caliber=5.69;
+        ACE_bulletLength=23.012;
+        ACE_bulletMass=4.0176;
+        ACE_ammoTempMuzzleVelocityShifts[]={-26.55, -25.47, -22.85, -20.12, -16.98, -12.80, -7.64, -1.53, 5.96, 15.17, 26.19};
+        ACE_ballisticCoefficients[]={0.307};
+        ACE_velocityBoundaries[]={};
+        ACE_standardAtmosphere="ASM";
+        ACE_dragModel=1;
+        ACE_muzzleVelocities[]={791, 897, 961};
+        ACE_barrelLengths[]={264.0, 393.7, 508.0};
+    };
+
+    CLASS(556x45_EPR_M855A1_Red): tacgt_556x45_EPR_M855A1 {
```

Comment:
And this is why `CLASS` macro shouldn't include `class` keyword in front. This should read:
```suggestion
    class CLASS(556x45_EPR_M855A1_Red): CLASS(556x45_EPR_M855A1) {
```